content/post/analytics_brain_trust.md

```diff
@@ -0,0 +1,86 @@
+---
+author: Scott Breitenother
+title: "Building your analytics brain trust"
```

Comment:
I prefer Title Case here

Reply:
Alternative Title:

> How I Learned to Be an Analytics Leader

Reply:
I'm going to think on the alternative title. I see pros and cons of using it